Server/OCPServerHardwareManagement.v1_1_0.json

```diff
@@ -209,11 +232,23 @@
 				}
 			}
 		},
-		"Thermal": {
+		"PowerSubsystem": {
+            "PropertyRequirements": {
+                "EnvironmentMetrics": {
+			        "ReadRequirement": "IfImplemented",
+			        "PropertyRequirements": {
```

Comment:
This needs to go into an EnvrionmentMetrics resource set of requirements...

But there's also no "EnvironmentMetrics" in PowerSubsystem...